src/seedu/addressbook/data/person/Printable.java

```diff
@@ -0,0 +1,15 @@
+package seedu.addressbook.data.person;
+
+public interface Printable {
+
+    public String getPrintableString();
+    
+    default public String getPrintableString(Printable...printables) {
+        StringBuilder printableString = new StringBuilder();
+        for (Printable object : printables) {
+            printableString.append(object.getPrintableString());
+        }
+        
+        return printableString.toString();
+    }
```

Comment:
This method shouldn't be here. Also, this method is never called. Discuss with your teammates.